configuring.tex

```diff
@@ -92,47 +97,53 @@ \section{Preparing for First Use: Formatting SD Cards}
-That means that the microSD card needs to have the essential files on it, otherwise MEGA65 cannot boot properly and will fall back into loading the OpenROM which does not support all the MEGA65 features.
-In general, if MEGA65 cannot boot properly and fall back to OpenROM, it will be shown in the power-up screen, similar to this:
+Straight out-of-the-box, MEGA65 will only have one internal SD card installed, accessible via the trap-door under the case. This SD card contains all essential files needed to properly boot up.
+If an external microSD card is also detected during boot up, the MEGA65 gives it higher priority and will try to boot from it.
+That means that the external microSD card needs to have the essential files on it, otherwise the MEGA65 will not boot up properly and will fall back to loading the OpenROM which does not support all MEGA65 features.
+In general, if your MEGA65 cannot boot properly and falls back to OpenROM, your power-up screen will look similar to this:
 
 \includegraphics[width=\linewidth]{images/mega65_OpenROM_boot_noSD.png}
 
 
 \section{Installing ROM and Other Support Files}
 \label{sec:installingrometc}
 
-The MEGA65 FDISK/FORMAT utility will install a version of the
-open-source OpenROM project's C64-compatible ROMs as part of the
-formatting process.
-The default ROM however is read from the file MEGA65.ROM, which contains
+The MEGA65 FDISK/FORMAT utility will add to your SD card a version of the
+open-source 'Open ROMs' project's C64-compatible ROM, and will name it MEGA65.ROM.
+
+For MEGA65 owners, we have replaced this file with the latest ROM from the 'Closed ROMs'
+project. It provides many improvements over original/incomplete C65 ROMs. It contains
 the operating system, BASIC 65, CBDOS and the machine language monitor BSMON.
 This ROM is developed especially for the MEGA65 and can be
 identified by the version number 92xxxx.
 
 However, you may have other ROMs that you wish to
-use on the MEGA65.
+use on your MEGA65.
 You can copy as many of these as you wish onto the
 SD card.  Make sure that they have the .ROM extension.  The default ROM
-should be called MEGA65.ROM.  These files
+should be called MEGA65.ROM. These files
 must be 128KB in size, and use the same internal format as ROMs
 intended for the C65.  This means that the C64-mode KERNAL must be
 placed at offset \$E000, a C65-mode BASIC at \$A000, and a suitable
 character set at \$D000.
 
+You can optionally name your alternate ROMs as 'MEGA65x.ROM', replacing 'x' with a number from 0 to 9.
+This allows you to quickly boot-up to your alternate ROMs by holding down a number from 0 to 9 prior 
+to powering on your MEGA65.
+
 Other important support files include FREEZER.M65 and AUDIOMIX.M65, which
-allow you to use the MEGA65's integrated freezer. More details provided below.
+allow you to use the MEGA65's integrated freezer. More details provided in the 'Floppy Disks And D81 Images' 
+chapter on page \pageref{cha:freezer}.
```

Comment:
More details _are_ provided

Reply:
Oops, missed this one, seems to have gotten hidden in the web interface, ok fixed this too.